tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -193,13 +193,43 @@ def testPluginIsActive(self):
     self._GenerateProjectorTestData()
     self._SetupWSGIApp()
 
-    # Embedding data is available.
+    patcher = tf.test.mock.patch('threading.Thread.start', autospec=True)
+    mock = patcher.start()
+    self.addCleanup(patcher.stop)
+
+    self.assertEqual(0, mock.call_count)
```

Comment:
This assertion is pretty useless: there's no chance for anyone to call `threading.Thread.start` between the time that you start the mock and this check. Might as well remove.